tools/system_libs.py

```diff
@@ -41,7 +41,7 @@ def read_symbols(path, exclude=None):
       symbols = filter(lambda symbol: symbol not in exclude, symbols)
     return set(symbols)
 
-  default_opts = []
+  default_opts = ['-Werror']
```

Comment:
A quick option here might be to enable the new `-Werror` flag in asm.js builds, and not for the wasm backend. Eventually it would be nice to enable it there too and not have warnings in system libs but that's not urgent in any way.